templates/templates/preview/model_plain_text.template

```diff
@@ -1,107 +1,97 @@
-        
-        {%- if model is defined -%}
-        Model name: {{ model.name }}
-            Executable:  
-            {%- for param in model.run_settings.exe %}
-                {{ param }}
+    
+    {%- if model is defined %}
+        Executable: {{ model.run_settings.exe[0] }}
+        Executable Arguments:
+        {%- for param in model.run_settings.exe_args %}
+          {{ param }}
+        {%- endfor %}
+        {%- if model.batch_settings -%}
+        Batch Launch: True
+        {% else %}
+        Batch Launch: False        
+        {%- endif -%} 
+        {%- if model.params %}        
+        Model Parameters: 
+        {%- for param, value in model.params.items() %}
+          {{ param }}: {{ value }}
+        {%- endfor -%}
+        {% endif %}
+        {%- if model.files %}
+        {%- if model.files.tagged %}
+        Tagged Files for Model Configuration: 
+          {%- for tagged in model.files.tagged %}
+          {{ tagged }} 
+          -> {{ model.path }}
+          {%- endfor %}
+          {%- endif -%}
+          {%- if model.files.copy %}
```

Comment:
super nitpicky, but if you have extra time (of course), consider fixing indentation so the `{%- if model.files.copy %}` is at the same level as the thing it's controlling (`Copy Files:...`). I think that might improve the readability.

Second, Consider the repetition in the tagged/copy/symlink sections here as a potential sub-template... You could pass the title and list source as args avoid having to change multiple places if this needs new formatting.

Reply:
fixed spacing